src/debug.c

```diff
@@ -895,20 +908,23 @@ void _serverPanic(const char *file, int line, const char *msg, ...) {
 
     bugReportStart();
     serverLog(LL_WARNING,"------------------------------------------------");
-    serverLog(LL_WARNING,"!!! Software Failure. Press left mouse button to continue");
-    serverLog(LL_WARNING,"Guru Meditation: %s #%s:%d",fmtmsg,file,line);
```

Comment:
:( i rather enjoyed those references

Reply:
i can revive them, just need to add another line with the redis server version.
or maybe we can skip it relying on the fact it's part of the INFO string?

Reply:
i vote for adding another line with the version.. in case the INFO part is missing from a user report / because of another bug

Reply:
If we're that paranoid, we can add the git sha too.. 
But actually unlike other info (e.g current client info, command stats, etc), we can always ask the person who submitted the report which version he's running, so I don't think we need to be that paranoid. 
The question is if for some reason it is needed at the very top (like the signal handler does)?

Reply:
reverted